src/components/Layout/Header/Header.tsx

```diff
@@ -43,19 +50,34 @@ export const Header = () => {
     return () => document.removeEventListener('keydown', handleKeyPress)
   }, [handleKeyPress])
 
+  const handleBannerClick = () => dispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
+
   return (
     <>
-      <Flex
-        height='4.5rem'
-        bg={bg}
-        borderBottomWidth={1}
-        borderColor={borderColor}
-        width='full'
-        position='sticky'
-        zIndex='banner'
-        top={0}
-      >
-        <HStack width='full' px={4}>
+      <Flex direction='column' bg={bg} width='full' position='sticky' zIndex='banner' top={0}>
+        {walletInfo?.deviceId === 'DemoWallet' && (
```

Comment:
`'DemoWallet` is used in multiple places and should definitely be a constant

Reply:
Since this is being set by the string in `DemoConfig`, I've just referenced that instead